src/egon/data/cli.py

```diff
@@ -99,6 +99,15 @@
     ),
     show_default=True,
 )
+@click.option(
+    "--household-demand-source",
+    type=click.Choice(["IEE", "demand-regio"]),
+    default="demand-regio",
+    help=(
+        "Choose the source to calculate and allocate household demands."
```

Comment:
It is not that easy to understand what IEE and demand-regio mean. I would either add some documentation about the difference and/or call it differently, eg.g "slp" oder "bottom-up-profiles".

Reply:
The required changes were done in f940f5026c646a64101adcb09a02bc583227bf0a